scripting/get5/recording.sp

```diff
@@ -0,0 +1,124 @@
+stock bool StartRecording() {
```

Comment:
nit: it probably makes more sense for this & StopRecording to be a `public` still (though practically it doesn't really matter). The only value of stock is that the function wouldn't get compiled into the plugin if unused (so it's typically reserved for "library" style files included by many plugins).